src/st/no4_psp/unk_5EE8.c

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "common.h"
+
+INCLUDE_ASM("st/no4_psp/psp/no4_psp/unk_5EE8", func_pspeu_0923D568);
```

Comment:
This is the start of first_c_file on PSX side. I'm pretty sure most if not all of these functions are consistent with that file and likely can be codeshared directly once it's all done